packages/hub/readme.md

```diff
@@ -39,13 +39,15 @@ $ yarn hub:watch (will rebuild app on file change)
 
 ### Interacting with the hub from a browser
 
+**NOTE**: it is important to define a unique `HUB_ADDRESS` if others are running the hub locally since all hub instances use the same Firebase realtime database.
+
 To play against the hub from the browser client, the hub and the browser need to:
 
-- Share the state-channel address of the hub. A good way to do so is to create a `.env.development.local` in the monorepo root with HUB_ADDRESS and HUB_PRIVATE_KEY defined.
-- Point to the same local Ganache server. Configure your `.env.*.local` files accordingly.
-- Point to the same contract addresses on Ganache.
+- Share the state-channel address of the hub. A good way to do so is to create a `.env.development.local` in the monorepo root with `HUB_ADDRESS` and `HUB_PRIVATE_KEY` defined.
+- Point to the same local Ganache server. Configure your `.env` files accordingly. This should work without any modifications.
```

Comment:
```suggestion
- Point to the same shared local Ganache server. This should work without any modifications. To see which ports are being used by the `hub` and `wallet`, and to verify they are the same, you can reference the `GANACHE_PORT` environment variable which by default is set in `.env.development` of each package.
```

Reply:
At the moment, ganache env vars are set in `.env` (and not `.env.development`) in all packages.

Reply:
Hopefully clearer in 04005dd79e9bb15826f6d186e91b33ba99d4a0d6